src/dali/plugin/input/rest/coinbase_pro.py

```diff
@@ -311,6 +311,7 @@ def _process_fills(self, transaction: Any, in_transaction_list: List[InTransacti
                 to_currency_size = from_currency_size * RP2Decimal(fill[_PRICE])
                 to_currency_price = usd_volume / to_currency_size
             elif fill_side == _BUY:
+                (from_currency, to_currency) = (to_currency, from_currency)
```

Comment:
this is the one line fix

Reply:
Nice, one liner fixes are my favorite :)